templates/jobs/base.html

```diff
@@ -1,6 +1,10 @@
 {% extends 'global/base.html' %}
 {% load staticfiles %}
 
+{% block extra_stylesheets %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/community.css' %}" media="screen">
+{% endblock extra_stylesheets %}
+
```

Comment:
wasn't there a bootstrap version 3.0.0 in jobs/base?  So we need it or is it OK with the global/base version?